src/trade.js

```diff
@@ -96,9 +96,9 @@ class Trade extends Exchange.Trade {
         // This log only happens if .set() is called after .debug is set.
         console.info('Trade ' + this.id + ' from Coinify API');
       }
-      this._createdAt = new Date(obj.createTime);
-      this._updatedAt = new Date(obj.updateTime);
-      this._quoteExpireTime = new Date(obj.quoteExpireTime);
+      this._createdAt = new Date(obj.createTime).getTime();
+      this._updatedAt = new Date(obj.updateTime).getTime();
+      this._expiresAt = obj.quoteExpireTime ? new Date(obj.quoteExpireTime).getTime() : new Date().getTime() - 1;
```

Comment:
What does the `-1` do? Is it to build a 1 second grace period? If so, I'm not sure if this is the most appropriate place to do that. But that's not a big issue.